universe/UniverseGenerator.h

```diff
@@ -42,7 +42,7 @@ class FO_COMMON_API MonsterFleetPlan : public FleetPlan {
 public:
     MonsterFleetPlan(const std::string& fleet_name, const std::vector<std::string>& ship_design_names,
                      double spawn_rate = 1.0, int spawn_limit = 9999, const Condition::ConditionBase* location = 0,
-                     bool lookup_name_userstring = false) :
+                     bool can_alter_starlanes = false, bool lookup_name_userstring = false) :
```

Comment:
Just to be sure: I guess you just missed that line when you removed the `CanAlterStarlanes` token and we don't need that additional parameter (`can_alter_starlanes`) here anymore, right?